src/SelectableBox/SelectableBoxSet.jsx

```diff
@@ -35,6 +36,7 @@ const SelectableBoxSet = React.forwardRef(({
         `pgn__selectable_box-set--${columns || DEFAULT_COLUMNS_NUMBER}`,
         className,
       ),
+      ...props,
```

Comment:
Actually, perhaps a quick clarifying question to verify with @wittjeff: Are the `aria-label` or `aria-labelledby` attributes _required_ from an a11y perspective, or optional but recommended?

If **required** in order to be generally a11y compliant as a standalone component, then I suppose we may want to consider enforcing the specific required a11y props (e.g., I wonder if there's a way to have both `aria-label` and `aria-labelledby` optional in the prop types but still conditionally required in some way). If were to enforce such an explicit prop as required, it would technically be a breaking for consumers, though there's not that many consumers of `SelectableBox` yet anyways, i.e. may need to release as a new major version if we were to make either of these props required.

All that said, I think we should go forward how you have it here for now; but would recommend getting that clarification from @wittjeff in terms of whether we should actually make it truly _required_ / enforced. We could also consider a middle ground where if either an `aria-label` or `aria-labelledby` is not provided, a `console.warn` is called, which consumers may see during local development as a hint to include these attributes.

Non-blocking feedback for this immediate PR, though!

Reply:
> That being said, I don't know if there is a way to require one OR the other within our components? 

@MaxFrank13 would likely need to write a custom prop type definition, e.g.: https://stackoverflow.com/a/46337223

[This file](https://github.com/openedx/paragon/blob/5dff07e45f779021e2a4ade0738b8b9fe48e3953/src/utils/propTypes/utils.js) in Paragon is a collection of a handful of re-usable prop types. Consider looking into whether `customPropTypeRequirement` might be applicable here?

Reply:
For div role="group" or div role="radiogroup" (the role should match whether we're using checkboxes or radio inputs), an accessible Name is a WCAG requirement (which in these cases would come from aria-label or aria-labelledby), so please make it a requirement here.

The only exception I can think of is if someone took our current control and wrapped it with a <fieldset> and <legend> (which are semantically equivalent to the above), but that seems exceedingly unlikely.

Reply:
> so please make it a requirement here.

Thanks @wittjeff!

@MaxFrank13 Given this, let's try to make explicit props for `aria-label` and `aria-labelledby`, where `aria-label` becomes required if `aria-labelledby` is not already set, possibly utilizing `customPropTypeRequirement` mentioned above. And, then keep the additional prop spreading in place since that adds support for attrs like `data-testid`.

Reply:
@MaxFrank13 If we do conditionally require one of these attributes, I would consider whether it needs to be released a breaking change (e.g., if a consuming MFE using `SelectableBox.Set` upgrades to this version, will any of their tests/CI begin failing?). My hunch is that it should be fine to release the required prop without a breaking change but worth verifying.

Reply:
I'm not clear on your logic with the last comment @adamstankiewicz . This would seem to be a breaking change, yes? I mean I suspect this error is probably out there. When I get back later I'll file issues with the pages that use this control without labels already.

Reply:
@wittjeff I guess I just wanted to do a quick sanity check that the prop type warnings do indeed fail CI (linting?), as opposed to simply logging a prop type warning in the console devtools that may be effectively ignored / not seen by engineers. You're right that it should and, if so, should be released as a breaking change.

Reply:
Thanks for the feedback, @wittjeff ! 

@adamstankiewicz that is a very useful file! Thanks for the tip. I didn't know we could do this with our prop types. 

From what I gather, this needs to be released as a breaking change. When it comes time to merge this, I will ask for a sanity check before I hit the big, scary green button. From the README, I gather that it is pretty straightforward but I would like to know if there is anything else that is maybe not included in there. I assume we need to alert the communication channels in some way?